esmvalcore/config/extra_facets/native6-mappings.yml

```diff
@@ -0,0 +1,192 @@
+# Extra facets for native6 data
+
+# Notes:
+# - All facets can also be specified in the recipes. The values given here are
+#   only defaults.
+
+# A complete list of supported keys is given in the documentation (see
+# ESMValCore/doc/quickstart/find_data.rst).
+---
+
+ERA5:
+
+  # Settings for all variables of all MIPs
+  '*':
+    '*':
+      family: E5
+      regrid: true
+      type: an
+      typeid: '00'
+      version: ''  # necessary to get a nice output file name
```

Comment:
```suggestion
      version: 'v1'  # necessary to get a nice output file name
```
may be nicer? This is also the version we use for NetCDF ERA5 data.

Reply:
Actually, I think this is already specified in the recipe

Reply:
Hmm, I am not sure about this. Does it really make sense to invent an arbitrary version number here? The only reason we need this is to derive the [output file name](https://github.com/ESMValGroup/ESMValCore/blob/4c3f1beb0e55fa70d0e7c9bdfcc1096231ecce12/esmvalcore/config-developer.yml#L106); it's not used at all to find the data.

Reply:
Yes, we invent version numbers for all observational and reanalysis datasets.

Reply:
This is certainly not true for all observational and reanalysis datasets. I will change it in the extra facets file..

Reply:
https://github.com/ESMValGroup/ESMValCore/pull/2178/commits/0fea4abd583b8f5fb5c1a6d1a2d300d6090a23ff